MonoGame.Framework/Audio/SoundEffectInstance.XAudio.cs

```diff
@@ -39,6 +39,12 @@ private void PlatformApply3D(AudioListener listener, AudioEmitter emitter)
             e.CurveDistanceScaler = SoundEffect.DistanceScale;
             e.DopplerScaler = SoundEffect.DopplerScale;
             e.ChannelCount = _effect._format.Channels;
+            //stereo channel
+            if(e.ChannelCount > 1 )
+            {
+                e.ChannelRadius = 0;
+                e.ChannelAzimuths = new float[] { 0f, 0f };
+            }
```

Comment:
cool, I think you are right for the PR, I'll clean it and also to the array, I did not want too impacted your work.
for the last point,"0" seems to be a good default value.
ChannelRadius is the distance from the Possition of imitter.
azimuths is value between 0 and 2pi, both value affects the least the nature of sound as in XNA.